docs/apache-airflow/installation/index.rst

```diff
@@ -0,0 +1,316 @@
+  files are managed by Apache Airflow release managers to make sure that you can repeatably install Airflow from PyPI with all Providers and
+  required dependencies.
+
+* In case of PyPI installation you could also verify integrity and provenance of the packages of the packages
+  downloaded from PyPI as described at the installation page, but software you download from PyPI is pre-built
+  for you so that you can install it without building, and you do not build the software from sources.
+
+**Intended users**
+
+* Users who are familiar with installing and configuring Python applications, managing Python environments,
+  dependencies and running software with their custom deployment mechanisms.
+
+**What are you expected to handle**
+
+* You are expected to install Airflow - all components of it - on your own.
+* You should develop and handle the deployment for all components of Airflow.
+* You are responsible for setting up database, creating and managing database schema with ``airflow db`` commands,
+  automated startup and recovery, maintenance, cleanup and upgrades of Airflow and Airflow Providers.
+
+**What Apache Airflow Community provides for that method**
+
+* You have :doc:`/installation/installing-from-pypi`
+  on how to install the software but due to various environments and tools you might want to use, you might
+  expect that there will be problems which are specific to your deployment and environment you will have to
+  diagnose and solve.
+* You have :doc:`/start/local` where you can see an example of Quick Start with running Airflow
+  locally which you can use to start Airflow quickly for local testing and development.
+  However this is just an inspiration. Do not expect this docker-compose is ready for production installation,
+  you need to build your own production-ready deployment in this approach.
+
+**Where to ask for help**
+
+* The ``#troubleshooting`` channel on Airflow Slack for quick general
+  troubleshooting questions. The `GitHub discussions <https://github.com/apache/airflow/discussions>`__
+  if you look for longer discussion and have more information to share.
+* If you can provide description of a reproducible problem with Airflow software, you can open
+  issue at `GitHub issues <https://github.com/apache/airflow/issues>`__
+
+
+Using Production Docker Images
+''''''''''''''''''''''''''''''
+
+More details: :doc:`docker-stack:index`
+
+**When this option works best**
+
+This installation method is useful when you are familiar with Container/Docker stack. It provides a capability of
+running Airflow components in isolation from other software running on the same physical or virtual machines with easy
+maintenance of dependencies.
+
+The images are build by Apache Airflow release managers and they use officially released packages from PyPI
+and official constraint files- same that are used for installing Airflow from PyPI.
```

Comment:
I feel all the "When this options works best", "Intended Users", "Expected to handle" can be summarized in a table instead of separate paragraph so it is easy to compare, or a separate section on "Which method of Installation should be used?"

Reply:
I tried. But our table layout is TERRIBLE - the table expands beyond available space and it looks REALLY bad.

Reply:
I will try to fix it though, because I also think table format would be much, much better